packages/devtools_app/test/shared/table_test.dart

```diff
@@ -1006,7 +1006,34 @@ void main() {
           ),
         ),
       );
+
       expect(find.byWidget(table), findsOneWidget);
+      expect(
```

Comment:
just to make sure, does this part of the test you added fail without the fix you added in this PR? Might be good to add a comment "Regression test for (<github issue link>)" to make that clear